firedantic/tests/tests_async/conftest.py

```diff
@@ -244,6 +265,9 @@ async def update_field(self, data) -> MockOperation:
         self.updated_field = data
         return MockOperation()
 
+    list_indexes = AsyncMock(return_value=MockListIndexOperation([]))
```

Comment:
Why do you define the `list_indexes` and `create_index` here and not as `self.` in the `__init__`? As a direct cause of this you now need to run `mock_admin_client.create_index.reset_mock()` as the first thing in many of your tests, which make the tests a bit more complex and increases the risk of making an error when making a new test case.

Reply:
it was a brain melting mistake, thanks